libs/langchain-azure-cosmosdb/src/azure_cosmosdb_nosql.ts

```diff
@@ -78,7 +78,7 @@ export interface AzureCosmosDBNoSQLConfig
   readonly metadataKey?: string;
 }
 
-const USER_AGENT_PREFIX = "langchainjs-azure-cosmosdb-nosql";
+const USER_AGENT_SUFFIX = "LangChain-CDBNoSQL-VectorStore-JavaScript";
```

Comment:
Could you explain why you made this change?
The naming is following a common pattern we have for all our JS integrations, which makes it easier to use filters: `<framework>-<integration_name>`
If we need a distinction, it would be best to use `<framework>-<integration>-<type>`, ie `langchainjs-azure-cosmosdb-nosql-vectorstore`

Reply:
@sinedied we are following a user agent pattern across all ai integrations, and we came up with "LangChain-CDBNoSQL-VectorStore-JavaScript"